float64 beta                        # [rad]
float64 dbeta                       # [rad/s]
float64 d2beta                      # [rad/s^2]
float64 d3beta                      # [rad/s^3]
float64 d4beta                      # [rad/s^4]
bool grasp_or_not
